proposesolution/gemini.go

```diff
@@ -0,0 +1,36 @@
+package proposesolution
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/google/generative-ai-go/genai"
+	"github.com/securego/gosec/v2/issue"
+	"google.golang.org/api/option"
+)
+
+func GetSolutionFromGemini(apiKey string, issues []*issue.Issue) {
+	if len(issues) == 0 || apiKey == "" {
```

Comment:
I would check the apikey and before calling this function, and if is empty skip the call all together. We can keep the issues check here.